extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/NarayanaQuarkusConfiguration.java

```diff
@@ -0,0 +1,147 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class NarayanaQuarkusConfiguration {
+    // non final properties
+    /**
+     * config
+     */
+    @ConfigItem(name = "CoordinatorEnvironmentBean.commitOnePhase", defaultValue = "true")
+    public boolean commitOnePhase;
+    /**
+     * config
+     */
+    @ConfigItem(name = "ObjectStoreEnvironmentBean.objectStoreDir", defaultValue = "PutObjectStoreDirHere")
+    public String objectStoreDir;
+    /**
+     * config
+     */
+    @ConfigItem(name = "ObjectStoreEnvironmentBean.transactionSync", defaultValue = "true")
+    public boolean transactionSync;
+
+    //    @ConfigItem(name = "CoreEnvironmentBean.nodeIdentifier", defaultValue = "1")
+    //    public String nodeIdentifier;
+    /**
+     * config
+     */
+    @ConfigItem(name = "JTAEnvironmentBean.xaRecoveryNodes", defaultValue = "1")
+    public List<String> xaRecoveryNodes;
+    /**
+     * config
+     */
+    @ConfigItem(name = "JTAEnvironmentBean.xaResourceOrphanFilterClassNames", defaultValue = "com.arjuna.ats.internal.jta.recovery.arjunacore.JTATransactionLogXAResourceOrphanFilter,"
+            + "com.arjuna.ats.internal.jta.recovery.arjunacore.JTANodeNameXAResourceOrphanFilter,"
+            + "com.arjuna.ats.internal.jta.recovery.arjunacore.SubordinationManagerXAResourceOrphanFilter")
```

Comment:
adding `SubordinateJTAXAResourceOrphanFilter` should be considered

Reply:
Again, I just duplicated our own defaults. If this property makes sense for quarkus then +1;
Is this property part of your analysis of the orphan filters that is being discussed on our forum?